pig/src/main/kotlin/org/partiql/pig/generator/kotlin/KTypeDomainConverter.kt

```diff
@@ -426,4 +429,9 @@ private class KTypeDomainConverter(
                 else -> this.typeName.snakeToPascalCase()
             }
         }
+
+    private fun TypeAnnotation.kotlinString(): String = when (this) {
```

Comment:
nit: may be `kotlinAnnotation`?
```suggestion
    private fun TypeAnnotation.kotlinAnnotation(): String = when (this) {
```

Reply:
I would expected `kotlinString` to return a `kotlin.String`, and `kotlinAnnotation` to return a [`kotlin.Annotation`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.annotation/).

Reply:
Now it makes more sense. Thanks!